tests/align/linear/gen_tensors.sh

```diff
@@ -4,6 +4,6 @@ eval "$(conda shell.bash hook)";
 rm align/linear/out/*.pt;
```

Comment:
why do we need to keep a folder for the `linear` and `layernorm` operators, but not the other ones? Also, why do we have the `gen_tensors.sh` script for the two operators above?

Reply:
the linear and layernorm operator have some bugs internal, 
I will create an issue for them. After fixed, I will move them into the main test function

Reply:
I see! In that case, since this is mainly a refactor, should we still port the linear/layernorm tests to the new file and just comment out the tests from the script until they are fixed?

Reply:
I think you are right, fixed